app/components/players/list.hbs

```diff
@@ -3,6 +3,15 @@
         <th {{on "click" (fn @onHeaderClick 'id')}}>id</th>
         <th {{on "click" (fn @onHeaderClick 'firstName')}}>First name</th>
         <th {{on "click" (fn @onHeaderClick 'lastName')}}>Last name</th>
+        <th>
+            Records on page
+            <PowerSelect
+                @options={{@limitValues}}
+                @selected={{@limit}}
+                @onChange={{@selectLimit}} as |limit|>
```

Comment:
What do you think about wrapping the closing tag to the new line?IMHO, it's kinda difficult to read in this format